package.json

```diff
@@ -15,6 +15,7 @@
     "react-bootstrap": "^2.7.1",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
+    "reactstrap": "^9.1.6",
```

Comment:
Why are we using react strap? 
There's a React Bootstrap component 
Here: https://react-bootstrap.netlify.app/components/offcanvas/#rb-docs-content